torch/fx/graph_module.py

```diff
@@ -35,16 +35,21 @@ def deserialize_graphmodule(root : torch.nn.Module, src : str) -> torch.nn.Modul
     to the original code. Then it symbolically traces through it to get the
     GraphModule
     """
-    root = copy.copy(root)
+    # We create a dummy class here because symbolic_trace pulls the forward()
+    # function off of the class, rather than the instance
+    class Dummy(torch.nn.Module):
+        def __init__(self, root, src):
+            super().__init__()
```

Comment:
nit: its a bit confusing to do class wide modifications in an instance. This would perhaps be clearer:
```
class Dummy(...):
   def __init__(self, root):
      self.root = root

Dummy.forward = generate_forward(src)
```

Also dummy is not a descriptive name, `CodeOnlyModule`? Not great either but it should describe what the object contains.